cng/hkdf.go

```diff
@@ -156,14 +156,19 @@ func ExtractHKDF(h func() hash.Hash, secret, salt []byte) ([]byte, error) {
 		return nil, errors.New("cng: unknown key data blob version")
 	}
 	// KEY_DATA_BLOB_VERSION1 format is:
-	// cbHash uint32 // Big-endian
-	// hashName [cbHash]byte
+	// cbHashName uint32 // Big-endian
+	// pHashName [cbHash]byte
 	// key []byte // Rest of the blob
```

Comment:
Where does this format come from? I'm not able to find it anywhere.

Reply:
Hmm, I don't remember where the definition came from. Maybe I just inspected the content of the data and figured it out? I'll ask the CNG team to document this payload, just as other algorithm payloads are.